protocol-units/bridge/contracts/src/AtomicBridgeInitiatorMOVE.sol

```diff
@@ -67,14 +66,10 @@ contract AtomicBridgeInitiatorMOVE is IAtomicBridgeInitiatorMOVE, OwnableUpgrade
         address originator = msg.sender;
 
         // Ensure there is a valid amount
-        if (moveAmount == 0) {
-            revert ZeroAmount();
-        }
+        require(moveAmount > 0, ZeroAmount());
 
         // Transfer the MOVE tokens from the user to the contract
-        if (!moveToken.transferFrom(originator, address(this), moveAmount)) {
```

Comment:
@andygolay As I explained in the issue, the scope of this change is _input validation_.
This `revert()` is not related to input validation, so it may not be changed to a `require`.